arrow/src/test/java/org/apache/calcite/adapter/arrow/ArrowAdapterTest.java

```diff
@@ -325,23 +325,29 @@ static void initializeArrowState(@TempDir Path sharedTempDir) throws IOException
         .explainContains(plan);
   }
 
+  /** <a href="https://issues.apache.org/jira/browse/CALCITE-6296">[CALCITE-6296]</a>. */
```

Comment:
We generally use this form to add Jira links for test cases (you can grep the codebase to see, if needed):
```suggestion
  /** Test case for
  * <a href="https://issues.apache.org/jira/browse/CALCITE-6296">[CALCITE-6296]
  * Support IS NULL in Arrow adapter</a>. */
```